### PR TITLE
45 current sons index

### LINKED ISSUE #45: Current list of approved SONs
As an authorized member of IT governance, I want to see the list of approved SONs.

Requirements
- [x] "Front end" style view of SONs list at /current-sons
- [x] Basic list view for sons at /sons
- [x] Sons list view removed from /home
- [x] "Statements of Need" link to /sons in Contributor Links menus
- [x] /home -> /dashboard + "Dashboard" link added to main navigation
- [x] I can search for SONs
- [x] A nicely formatted list view of statements of need (https://it.cornell.edu/itgovernance-project-management/3-current-it-statements-need-and-it-project-charters-access)

### PR DESCRIPTION
**Pull request by philwilliammee**
closes #45 

A SONS list page with tones styling, and simple text search. Very much like the current list on itc.

### CODE REVIEW

**Comment by woodseowl** [comment]
@philwilliammee Should I be reviewing this yet? I'm seeing new commits coming in today.

**Comment by philwilliammee** [comment]
Yeah I'm done. I also added tones styling to the main form. @jmh222 can review and add the styling that is missing from the original html_son in box

**Comment by philwilliammee** [comment]
updated with new requirements in #45 renames home to dashboard moves sons.index to sons..

File: routes/web.php [hunk 1]
```diff
@@ -52,8 +52,9 @@ function () {
         Route::group(
             ['middleware' => ['role:admin|pmo|contrib|auth']],
             function () {
-                Route::get('/home', 'HomeController@index')->name('home');
+                Route::get('/dashboard', 'HomeController@index')->name('dashboard');
```
Comment (woodseowl) [inline routes/web.php:55]: I'm wondering if this is why I am getting a 404? After I add/edit a SoN it is redirecting me to '/home' and returning a 404.

Comment (philwilliammee) [reply routes/web.php:55]: I'm pretty sure this is fixed in #49 and #50

File: resources/views/layouts/site-header.blade.php [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <nav class="dropdown-menu" id="main-navigation" aria-label="Main Navigation">
     <div class="container-fluid">
-        <a id="mobile-home" href="#"><span class="sr-only">Home</span></a>
+        <a id="mobile-home" href="#"><span class="sr-only">Dashboard</span></a>
```
Comment (philwilliammee) [inline resources/views/layouts/site-header.blade.php:3]: this shouldn't be a link, remove this

**Comment by philwilliammee** [comment]
@woodseowl  
these issues have been fixed in PR#49 and 50
changed from
https://github.com/CU-CommunityApps/cd-son/blob/66e2c59226decfe303453b50b924017501a42b28/app/Http/Controllers/SonController.php#L171 

to 
https://github.com/CU-CommunityApps/cd-son/blob/30b85964b746d4eb50c26df3ceda693afc7d9207/app/Http/Controllers/SonController.php#L158

**Comment by philwilliammee** [comment]
Issues with adding full coverage to unit testing routes addressed in issue #53

**Comment by philwilliammee** [comment]
ok let me know when you want to talk

**Pull request merged**